### DISCUSSION TITLE
Set fallback/default icon for element-icon?

### DISCUSSION THREAD

**Discussion by shivanshu-semwal** [davatorium/rofi#1489] [Theming] [answered]
I have some desktop file which don't have some suitable icon. I know i can fix the `.desktop` file to show icon. But still is there any way to set the icon if it is not present.

![image](https://user-images.githubusercontent.com/53912074/135622132-9627bcc4-ec13-4056-8826-df4579277b7e.png)

**Comment by DaveDavenport** [comment, marked as answer]
Yes, this was introduced in 1.7.0.
See [release notes](https://github.com/davatorium/rofi/discussions/1384)/[manpage](https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown).

**Reply by shivanshu-semwal** [reply]
So `fallback-application-icon: "/home/totoro/hi.png";` i tried something like this. It gave me error 

`** (process:160690): WARNING **: 18:40:56.316: /home/totoro/.config/rofi/rofi-launchers/default.rasi:8:27: Option: fallback-application-icon is not found.`

```
configuration {
    font: "Iosevka Nerd Font:style=Bold 12";
    show-icons: true;
    display-drun: "";
    drun-display-format : "{name}";
    disable-history: false;
    sidebar-mode: false;
    icon-theme: "Papirus";
    fallback-application-icon: "/home/totoro/hi.png";
}
```
I put it inside `configuration` block. What am i doing wrong?

**Reply by DaveDavenport** [reply]
what version of rofi are you using?

**Reply by shivanshu-semwal** [reply]
Thanks for the fast response, i'm using `1.7.0` version from the release. 
I did `-dump-config` thing and i found that it is using `application-fallback-icon` so I used that one and it works. 

But in manual this is mentioned:

```
-fallback-application-icon

Specify an icon to be used when the application icon in run/drun are not yet loaded or is not available.
```
[this one](https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown)

**Reply by DaveDavenport** [reply]
aah I will fix that, thanks.
